modules/gdnative/godot/string.cpp

```diff
@@ -960,38 +959,51 @@ uint32_t GDAPI godot_string_hash_utf8_chars_with_len(const wchar_t *p_str, godot
 	return String::hash(p_str, p_len);
 };
 
-godot_string GDAPI godot_string_md5_text(const godot_string *p_self) {
+godot_pool_byte_array GDAPI godot_string_md5_buffer(const godot_string *p_self) {
```

Comment:
Hey @karroffel !
I tried to implement the two methods, `md5_buffer` and `sha256_buffer`, that return `godot_pool_bytes_array`.

Could you please tell me if I made some mistakes ?

Thanks in advance :smile: